### ISSUE TITLE
PickList: Source and target listbox missing aria-label

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#17453]
### Describe the bug

[In version 17, listboxes got an aria label from the headers of the target and source elements](https://github.com/primefaces/primeng/blob/v17/src/app/components/picklist/picklist.ts#L150). Now these headers no longer exist, and therefore listboxes don't get aria labels. You should be able to set your own aria labels for them or add them from the configuration.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/picklist

### Environment

https://primeng.org/picklist

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Open [primeNg.org/picklist](https://primeng.org/picklist) 2. Inspect elements

### Expected behavior

Aria-labels fixed
